assets/facets.js

```diff
@@ -147,47 +156,34 @@ class FacetFiltersForm extends HTMLElement {
   }
 
   static renderCounts(source, target) {
-    const filterIsDrawer = Boolean(document.querySelector('.facets-container-drawer'));
-    const targetElement = filterIsDrawer
-      ? target.querySelector('.mobile-facets__list')
-      : target.querySelector('.facets__selected');
-    const sourceElement = filterIsDrawer
-      ? source.querySelector('.mobile-facets__list')
-      : source.querySelector('.facets__selected');
-
-    const targetElementAccessibility = filterIsDrawer && target.querySelector('.facets__summary');
-    const sourceElementAccessibility = filterIsDrawer && source.querySelector('.facets__summary');
-
-    if (sourceElement && targetElement) {
-      const currentActiveID = document.activeElement.id;
-      const isShowingMore = Boolean(target.querySelector('show-more-button .label-show-more.hidden'));
+    const targetSummary = target.querySelector('.facets__summary');
+    const sourceSummary = source.querySelector('.facets__summary');
 
-      if (sourceElement && targetElement) {
-        targetElement.outerHTML = sourceElement.outerHTML;
-      }
+    if (sourceSummary && targetSummary) {
+      targetSummary.outerHTML = sourceSummary.outerHTML;
+    }
 
-      if (!filterIsDrawer) {
-        const targetWrapElement = target.querySelector('.facets-wrap');
-        const sourceWrapElement = source.querySelector('.facets-wrap');
-        if (sourceWrapElement && targetWrapElement) {
-          if (isShowingMore) {
-            sourceWrapElement
-              .querySelectorAll('.facets__item.hidden')
-              .forEach((x) => x.classList.replace('hidden', 'show-more-item'));
-          }
+    const targetWrapElement = target.querySelector('.facets-wrap');
+    const sourceWrapElement = source.querySelector('.facets-wrap');
 
-          targetWrapElement.outerHTML = sourceWrapElement.outerHTML;
-        }
+    if (sourceWrapElement && targetWrapElement) {
```

Comment:
Why do we need to remove the hidden class for the show more item elements in the `renderCounts` function? 👀